src/Tvheadend.cpp

```diff
@@ -73,6 +69,15 @@ void CTvheadend::Start ( void )
   m_conn->Start();
 }
 
+void CTvheadend::Stop ( void )
+{
+  for (auto *dmx : m_dmx)
+    dmx->Close();
```

Comment:
Shouldn't we also do a `delete`?

Reply:
No, delete's are only done in the dtor. Stop just "resets".

Reply:
Of course